src/components/Section/SectionView.jsx

```diff
@@ -13,96 +13,92 @@ export class SectionViews extends React.Component {
   render() {
     // Iterate through child <SectionView /> components and check their props
     const children = React.Children.map(this.props.children, child => {
-      const currentSection = this.props.current || ''
-      // If the current route name matches one of the section view child component names
-      if (currentSection === child.props.name) {
-        let previousButton = <div className="btn-cell" />
-        if (child.props.back) {
-          const backtalk = `Go to previous section ${child.props.backLabel}`
-          previousButton = (
-            <button
-              className="btn-cell back"
-              title={backtalk}
-              aria-label={backtalk}
-              onClick={this.handleTransition.bind(this, child.props.back)}>
-              <div className="actions back">
-                <div className="icon">
-                  <i className="fa fa-arrow-circle-left" aria-hidden="true" />
-                </div>
-                <div className="text">
-                  <div className="direction">Back</div>
-                  <div className="label">{child.props.backLabel}</div>
-                </div>
+      let previousButton = <div className="btn-cell" />
```

Comment:
TBD - need to add this back into each section

Reply:
This is now completed at https://github.com/18F/e-QIP-prototype/pull/1431/files#diff-e0df189d16e0e8e1b2360e055bb7bd57R54